File: Asocioacion/python/primerEjercicio/Cliente.py
class Cliente:
    c = 0
    def __init__(self, nombreCliente):
        self.nombreCliente = nombreCliente
        self.mediosPago = []
        self.__class__.c+=1

    #Nombre del Cliente

    def getNombreCliente(self):
        return self.nombreCliente
    
    def setNombreCliente(self,nombreCliente):
        self.nombreCliente = nombreCliente
        
    #Metodo de agregacion y visualizacion de metodos de pago
        
    def agregarMetodo(self,medioPago):
        self.mediosPago.append(medioPago)

    def getMediosPago(self):
        return self.mediosPago

File: Asocioacion/php/primerEjercicio/Cliente.php
<?php

class Cliente{

    private string $nombreCliente;
    private $mediosPago = [];

    public function __construct(string $nombreCliente){

        $this->nombreCliente = $nombreCliente;

    }

    //Nombre del cliente

    public function getNombreCliente(){
        return $this->nombreCliente;
    }

    public function setNombreCliente($nombreCliente){
        $this->nombreCliente = $nombreCliente;
    }

    //Metodo para agregar y ver  metodos de pago
    public function agregarMedioPago($medio){
        array_push($this->mediosPago,$medio);
    }

    public function verMediosPago(){
        return $this->mediosPago;
    }

}

?>